src/main/java/vic/cinema/mapper/UserMapper.java

```diff
@@ -0,0 +1,17 @@
+package vic.cinema.mapper;
+
+import org.springframework.stereotype.Component;
+import vic.cinema.dto.UserResponseDto;
+import vic.cinema.model.User;
+
+@Component
+public class UserMapper {
+
+    public UserResponseDto getUserResponseDto(User user) {
```

Comment:
```suggestion
    public UserResponseDto toDto(User user) {
```
make your mappers' method names uniform

Reply:
Done